src/webhooks/webhooks.ts

```diff
@@ -0,0 +1,111 @@
+import { Webhook } from './interfaces/webhook.interface';
+import crypto from 'crypto';
+import { SignatureVerificationException } from '../common/exceptions';
+
+export class Webhooks {
+  constructEvent({
+    payload,
+    sigHeader,
+    secret,
+    tolerance = 180,
+  }: {
+    payload: any;
```

Comment:
We should consider using `unknown` instead of `any` in this public type signature.

Reply:
Thanks Marshall! The above issues should now be addressed.